homeassistant/components/device_tracker/ubus.py

```diff
@@ -92,8 +95,8 @@ def scan_devices(self):
         return self.last_results
 
     def _generate_mac2name(self):
-        """Must be implemented depending on the software."""
-        raise NotImplementedError
+        """Overridden if DHCP server is selected."""
```

Comment:
Given this being the docstring for that method, I suppose it should describe what it does and not just that it's overridden whenever necessary. Otherwise this patch seems straightforward and I'm ok to merge it.

Reply:
Amended to be more descriptive.

Reply:
Thanks!